benchmarks/inference-server/text-generation-inference/manifest-templates/text-generation-inference.tftpl

```diff
@@ -51,7 +51,7 @@ spec:
         - name: text-generation-inference
           ports:
             - containerPort: 80
-          image: "ghcr.io/huggingface/text-generation-inference:1.4.2"
+          image: "us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-text-generation-inference-cu121.2-2.ubuntu2204.py310"
```

Comment:
@alvarobartt this is a major upgrade for TGI (eg. 1.4.2 -> 2.2 per docs here, https://huggingface.co/blog/llama31-on-vertex-ai#3-register-the-meta-llama-31-405b-model-on-vertex-ai)

OOC, how have you tested this for the benchmarking suite?

Reply:
Hi @annapendleton, yes indeed the upgrade comes with performance improvements and support for more architectures; and indeed TGI < 2.0 is no longer the recommended approach; anyway those models should work normally with either version.

Also w.r.t. benchmarking we've done extensive benchmarking on TGI globally yes! Or do you mean with any suite you may have?

Happy to help with more information on any of those if needed! 🤗

Reply:
Alternatively, TGI 1.4 is also available within the Hugging Face DLCs on Google Cloud at us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-text-generation-inference-cu121.1-4.ubuntu2204.py310; but as mentioned, not recommended, anyway worst case scenario we can pin that URI for the mentioned example if needed.

Reply:
Ah I just mean have you tested that the ai-on-gke/tree/main/benchmarks/inference-server/text-generation-inference works with the upgraded image :) I wasn't sure if there were any flag / use changes between the major versions that would break what is already supported in ai-on-gke/benchmarks suite. We don't have any automated tests so it requires manual validation. From what I've read, it sounds like this should work as a drop in replacement?

Reply:
Oh yes sorry I didn't get that! Yes, should work as is, no need to change anything else! Anyway I believe we can still re-run the benchmarking suite on your end, is that something you can do, should we run that on our end? cc'ing also @brandonroyal here just in case there's anything we can help!

Reply:
@alvarobartt I won't have time to validate this as I'll be leaving for vacation in a few days - is there a desired deadline on when we upgrade the ai-on-gke/benchmarks to this image?

If it's not urgent, I would recommend we remove the upgrade in ai-on-gke/benchmarks from this PR so you can submit for the other related tutorials, and our team can take this task on separately to upgrade and validate (and also thinking we might want to parameterize this so users can specify which image they want to benchmark, and make this image the default image).

Reply:
Fair, I'll leave this comment open so that we can come back to this once you're back from vacation maybe and see if we can also update the container URI used for benchmarking? Thanks for the information 🤗

Reply:
Sounds good to me! Thanks for your flexibility. I also opened an issue for tracking, so definitely feel free to close this comment and submit this PR for the other related tutorials :)

ref: https://github.com/GoogleCloudPlatform/ai-on-gke/issues/821